CHANGES.rst

```diff
@@ -163,6 +163,9 @@ source_catalog
 
 - Updated step arguments in the documentation. [#4723]
 
+- Added fallback background estimation method to make background
```

Comment:
This should go into a new source_catalog entry up at the top of the file, under version 0.16.1 (not 0.16.0, which was already released).

Reply:
Fixed, thanks!